pom.xml

```diff
@@ -129,7 +133,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-gpg-plugin</artifactId>
-            <version>1.5</version>
+            <version>1.6</version>
```

Comment:
Do we use the `maven-gpg-plugin` without the profile? If not, let's factor it out of the main section of the POM. If so, I think a property might make sense.